src/main/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStepExecution.java

```diff
@@ -25,18 +26,6 @@
 
     private static final long serialVersionUID = 1L;
 
-    @StepContextParameter
```

Comment:
Note [migration guide](https://javadoc.jenkins.io/plugin/workflow-step-api/org/jenkinsci/plugins/workflow/steps/AbstractStepExecutionImpl.html#AbstractStepExecutionImpl--) though this need not be done now (can be a follow-up).

Reply:
I will follow up in separate PR.

Reply:
Never mind, adding it to this PR, seems like there is no good way of making dependent PRs on github.